src/main/java/controller/login/LoginController.java

```diff
@@ -0,0 +1,26 @@
+package controller.login;
+
+import use_case.login.LoginInputBoundary;
+import use_case.login.LoginInputData;
+
+/**
+ * The controller for the Login Use Case.
+ */
+public class LoginController {
+
+    private final LoginInputBoundary loginInputBoundary;
+
+    public LoginController(LoginInputBoundary loginInputBoundary) {
+        this.loginInputBoundary = loginInputBoundary;
+    }
+
+    /**
+     * Executes the Login Use Case.
+     * @param email the email of the user logging in
+     * @param password the password of the user logging in
+     */
+    public void execute(String email, String password) {
+        final LoginInputData loginInputData = new LoginInputData(email, password);
```

Comment:
If you pull the latest changes from upstream, you'll see that there are two errors thrown in the login use case: `AccountDoesNotExist` and `IncorrectPassword`. You can catch it here in this PR (you don't need to do anything about it, just catch it in a try catch block)